File: OrderAnalysis-ordergoods-interface/src/main/java/com/orderanalsis/ordergoods/service/DateCityGoodsCountService.java
package com.orderanalsis.ordergoods.service;
import java.util.List;
import com.orderanalsis.pojo.TbDateCityGoodsCount;

import entity.PageResult;
/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface DateCityGoodsCountService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<TbDateCityGoodsCount> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageResult findPage(int pageNum,int pageSize);
	
	
	/**
	 * 增加
	*/
	public void add(TbDateCityGoodsCount dateCityGoodsCount);
	
	
	/**
	 * 修改
	 */
	public void update(TbDateCityGoodsCount dateCityGoodsCount);
	

	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public TbDateCityGoodsCount findOne(Long id);
	
	
	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(Long [] ids);

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageResult findPage(TbDateCityGoodsCount dateCityGoodsCount, int pageNum,int pageSize);
	
}

File: OrderAnalsis-ordergoods-web/src/main/webapp/js/service/dateCityGoodsCountService.js
//服务层
app.service('dateCityGoodsCountService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../dateCityGoodsCount/findAll.do');		
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../dateCityGoodsCount/findPage.do?page='+page+'&rows='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../dateCityGoodsCount/findOne.do?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../dateCityGoodsCount/add.do',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../dateCityGoodsCount/update.do',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../dateCityGoodsCount/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../dateCityGoodsCount/search.do?page='+page+"&rows="+rows, searchEntity);
	}    	
});
